### ISSUE TITLE
Suitable Infrastructure not triggering as Manutech (Prelude 2)

### ISSUE THREAD

**Issue by gringo-lam** [terraforming-mars/terraforming-mars#7140]
Please include:
1. A clear and concise description of the bug.

Playing as Manutech with Suitable Infrastructure doesn't trigger the 2MC rebait effect
after playing multiple cards which increase a production no rebait was triggered (Mohole Area, Industrial Microbes, Building a Colony on Luna)

3. Clear steps to reproduce, and the expected behavior.

 Playing as Manutech with Suitable Infrastructure and playing these cards 
Luna Colony was built via Boards of Directors and playing Strategic Base Planning

5. Any screenshots
![tfm app bug3](https://github.com/user-attachments/assets/5d134dd0-5b70-437f-985f-8ac788db2d42)
![tfm app bug2](https://github.com/user-attachments/assets/0e524ebc-f63a-41e1-9a21-af90044e8451)
![tfm app bug](https://github.com/user-attachments/assets/6c66cc67-3059-4142-8822-2dec1220df4f)

7. Any information from  the Bug Report Dialog
I don't know where to pull this.

game ID = g851c8ba2a5e0
player ID = pbe7bbb075b48

**Comment by kberg** [comment]
Are you saying that it did not give the money because the log doesn't say so? Not everything is logged. If you are certain that you did not get the money because the MC balance was wrong, I will look further.

**Comment by gringo-lam** [comment]
Correct, it didn’t give the money.I played mohole lake and spent all money and steel to affordI ended with 0MC after the action although it should have been 2MC after the action.Kind regards Lukas Sent from my iPhoneOn 19. Dec 2024, at 06:47, Robert Konigsberg ***@***.***> wrote:﻿
Are you saying that it did not give the money because the log doesn't say so? Not everything is logged. If you are certain that you did not get the money because the MC balance was wrong, I will look further.

**Comment by ryannewton** [comment]
I tried to repro via playing Suitable Infrastructure + Manutech, and I got the 2MC bonus.

It doesn't log either stock bonus though, just the direct prod increase effect of the card I played (Import of Advanced GHG).
![Screenshot 2025-01-01 at 11 40 57 PM](https://github.com/user-attachments/assets/48d74adb-1da9-4fb2-854d-c5e24909490b)

player ID = p3d7bb3bb7fa9

Steps taken (that are relevant):

- Played Manutech
- Played Suitable Infrastructure
- Status: 32MC
- Played Import of Advanced GHG
- Status: 25MC = 32 (starting) - 9 (card cost) + 2 (SI effect); also 2 heat (Manutech effect)

**Comment by Borbarad2** [comment]
Could this issue now be closed? Is the issue solved?
